dataset/tinysnb/schema.cypher

```diff
@@ -1,5 +1,6 @@
 create node table person (ID INt64, fName StRING, gender INT64, isStudent BoOLEAN, isWorker BOOLEAN, age INT64, eyeSight DOUBLE, birthdate DATE, registerTime TIMESTAMP, lastJobDuration interval, workedHours INT64[], usedNames STRING[], courseScoresPerTerm INT64[][], PRIMARY KEY (ID));
 create node table organisation (ID INT64, name STRING, orgCode INT64, mark DOUBLE, score INT64, history STRING, licenseValidInterval INTERVAL, rating DOUBLE, PRIMARY KEY (ID));
+create node table movies (ID INT64, name STRING, PRIMARY KEY (ID));
```

Comment:
I think this dataset is only used to test utf-8 strings. To make the testing simpler, you can have one simply column name and use name as primary key.

Reply:
Yes previously ID didn't support chars so had to add that.
I think 2 columns are fine, or else I'll have to go back and change all the tests for this.

Reply:
We always use the minimum code/dataset for testing, so it is better for you to just keep one column in movies table.

Reply:
I've dropped the extra column, now name is the only column and primary key.